{
  "Launchpad": "https://bugs.launchpad.net/bugs/1223452",
  "Id": "1223452",
  "Gerrit": "https://review.openstack.org/176871",
  "IdGerrit": "176871",
  "IdCommit": "4812617002ee6a5044096e253413d6718003bca3",
  "Files": "nova/scheduler/filters/trusted_filter.py,nova/tests/scheduler/test_host_filters.py",
  "Lines_Inseted": "2",
  "Lines_Deleted": "7",
  "CommitParent": "234c6e0d6ab40846fbb05b6f48dec3c9da760f12",
  "Classification": "Unknown",
  "KeywordsTitle": "checks, trust level",
  "Description": "uses, will be checked, works, must be checking, will be registered, attestation server.",
  "KeywordsCommit": "different, remote attestation",
  "Comments": "I'm not sure, they need checks that the name of the node in a VM is the same that the hostname inthe hypervisor, but they weren't describing any failure, "
}
